.circleci/config.yml

```diff
@@ -10,6 +10,7 @@ defaults: &defaults
 run_test: &run_test
   steps:
     - checkout
+    - run: echo "export GIT_SHA1=${CIRCLE_SHA1}" >> $BASH_ENV
```

Comment:
I'm using this environment variable in `stable/python/Makefile` to refer to files from the pull-request rather than fixed files from `master` branch.